package.json

```diff
@@ -100,9 +102,11 @@
     "update-icons": "node ./scripts/update-icons",
     "copy": "copyfiles package.json README.md dist; yarn copy:rest",
     "copy:rest": "yarn copyfiles -u 1 'src/**/*.md' dist",
-    "release": "yarn clean && yarn build && yarn copy && CI=1 npm publish dist/ --access public",
-    "release:rc": "yarn clean && yarn build && yarn copy && CI=1 npm publish --tag rc dist/",
-    "prepare": "node scripts/husky-install || node ../scripts/husky-install"
+    "release": "yarn clean && yarn build && yarn copy && CI=1 changeset publish",
+    "prerelease:end": "changeset pre exit || :",
+    "prerelease:start": "changeset pre enter next || :",
+    "prepare": "node scripts/husky-install || node ../scripts/husky-install",
+    "version": "yarn update-icons && git add -A src && changeset version"
```

Comment:
One issue we found was that when doing a release, the package-lock version does not get changed. By doing npm i --package-lock-only this will regenerate the package-lock version. That said, I don't know if theres an equivalent or similar problem for yarn.

Reply:
Does changeset changes the version of the lock file?